myst_nb/core/config.py

```diff
@@ -64,7 +64,7 @@ def ipywidgets_js_factory() -> Dict[str, Dict[str, str]]:
             "crossorigin": "anonymous",
         },
         # Load IPywidgets bundle for embedding.
-        "https://unpkg.com/@jupyter-widgets/html-manager@^0.20.0/dist/embed-amd.js": {
+        "https://cdn.jsdelivr.net/npm/@jupyter-widgets/html-manager@1.0.6/dist/embed-amd.js": {
```

Comment:
can we add a comment here that provides a loose guidelines for when/how to update this import link? Or maybe just a block comment at the top of the `#Load IPywidgets..` section?

Reply:
Hehe, we're on the same wavelength: https://github.com/executablebooks/MyST-NB/pull/491/commits/2deaa406b91f1f6f7e93db27ef9c9befc41fb053